### PR TITLE
Doc - Ports to open and reviewing TPD/TDS related role readme files

### PR DESCRIPTION
**Pull request by florentlb**
Hi guys, I've started reviewing what you have already pushed in Readme files for TDP and TDS, and the root Readme. 
Please review my changes. Ping me if you have questions/remarks.

### CODE REVIEW

File: ansible/roles/sjs/README.md [hunk 1]
```diff
@@ -1,43 +1,32 @@
...
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
+//
 
 Dependencies
 ------------
 
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
-You need the role : 'java' and 'talend-repo'
+The following roles must be used to successfully install and deploy Talend Administrator Center:
```
Comment (sgolovnya) [inline ansible/roles/sjs/README.md]: Please update "Talend Administrator Center" with "Spark Job Server" (as this is for SJS), the rest is fine here (the list of roles required).

File: ansible/roles/streamsrunner/README.md [hunk 2]
```diff
@@ -1,43 +1,32 @@
...
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
+//
 
 Dependencies
 ------------
 
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
-You need the role : 'java' and 'talend-repo'
+The following roles must be used to successfully install and deploy Talend Administrator Center:
```
Comment (sgolovnya) [inline ansible/roles/streamsrunner/README.md]: Please update "Talend Administrator Center" with "Streamsrunner", the rest if fine here.

File: ansible/roles/tcomp/README.md [hunk 3]
```diff
@@ -1,43 +1,32 @@
...
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
+//
 
 Dependencies
 ------------
 
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
-You need the role : 'java' and 'talend-repo'
+The following roles must be used to successfully install and deploy Talend Administrator Center:
```
Comment (sgolovnya) [inline ansible/roles/tcomp/README.md]: "Talend Administrator Center" -> "Talend Components Catalog"

File: ansible/roles/tdp/README.md [hunk 4]
```diff
@@ -1,43 +1,32 @@
...
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
+//
 
 Dependencies
 ------------
 
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
-You need the role : 'java' and 'talend-repo'
+The following roles must be used to successfully install and deploy Talend Administrator Center:
```
Comment (sgolovnya) [inline ansible/roles/tdp/README.md]: "Talend Administrator Center" -> "Talend Data Preparation"

**Comment by florentlb** [comment]
Changes done

**Pull request merged**